### ISSUE TITLE
magick rock not found

### ISSUE THREAD

**Issue by sdifru** [3rd/image.nvim#194]
This plugin has been working great on my computer. Today I updated this plugin and get this error: `image.nvim: magick rock not found, please install it and restart your editor. Error: "module 'magick' not found:`.

How do I solve this problem?

**Comment by pr4th4m** [comment]
This might help

```
By default, brew installs into a weird location, so you have to add $(brew --prefix)/lib to DYLD_LIBRARY_PATH by adding something like export DYLD_LIBRARY_PATH="$(brew --prefix)/lib:$DYLD_LIBRARY_PATH" to your shell profile (probably .zshrc or .bashrc)
```
https://github.com/3rd/image.nvim/blob/master/README.md?plain=1#L187

**Comment by sdifru** [comment]
@pr4th4m I don't use mac. I use arch linux.

**Comment by allpandasarecute** [comment]
@sdifru, this is because you need to install `magick` package from `luarocks`
But it's only available for lua 5.1
So, if you use arch, you should install `lua51` and `luarocks` packages from pacman, then run `luarocks --local --lua-version=5.1 install magick`
After that, add this to your neovim config (using `lazy.nvim` here):
```lua
{
		"3rd/image.nvim",
		config = function()
			require("image").setup()
			package.path = package.path .. ";" .. vim.fn.expand("$HOME") .. "/.luarocks/share/lua/5.1/?/init.lua"
			package.path = package.path .. ";" .. vim.fn.expand("$HOME") .. "/.luarocks/share/lua/5.1/?.lua"
		end,
	}
```

**Comment by sdifru** [comment]
It works @allpandasarecute !
